tasks/upload-sbom-to-atlas/upload-sbom-to-atlas.yaml

```diff
@@ -4,20 +4,19 @@ kind: Task
 metadata:
   name: upload-sbom-to-atlas
   labels:
-    app.kubernetes.io/version: "0.1.0"
+    app.kubernetes.io/version: "0.3.0"
```

Comment:
As mentioned in the README, I think 0.2.0 makes more sense. It is okay to have multiple changes in one new version